vero-git-hub/notes-app
# Add note and archive functionality, improve note count tracking

Comprehensive update of an existing layout. Notes-app is an application for creating, managing and count notes into various categories. Overview of changes:

### New Features:

**Note Table:** create notes with preset categories, edit and delete. Automatic time setting, date parsing from the "Content" field.

**Archive functionality:** The app allows users to archive notes. Clicking the box icon on a note in the main table moves it to the "Archived Notes Table". Unpacking of archived notes is also available.

**Archive Note Table:** The table "Archive Note Table" has been added to display archived notes. Users can toggle the visibility of this table by clicking the eye icon next to "Archived."

**Count of Notes:** Display the count of notes for each category. The "Count Table" shows the count of both active and archived notes for each category.

### User Interface Enhancements:

**Hidden Message:** An informative message is displayed when the "Archive Note Table" is hidden, guiding the user on how to access it.

### How to Test:

1. Clone the repository and set up the project locally.
2. Open the index.html file in your preferred web browser.
3. Interact with the application to add, edit, archive, and delete notes.
4. Verify that the "Count Table" accurately reflects the counts for each note category in both the main table and the "Archive Note Table."
5. Toggle the visibility of the "Archive Note Table" using the eye icon and observe the hidden message behavior.

These updates are intended to provide users with a more organized note-taking experience.
